src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -86,9 +87,10 @@ public void proceed(JsonObject payload, ExportResult exportResult) {
     }, this::handleExportResult);
   }
 
-  protected void initBlocking(JsonObject request, JsonObject requestFileDefinitionJson, JsonObject mappingProfileJson, Map<String, String> params) {
+  protected void initBlocking(JsonObject request, JsonObject requestFileDefinitionJson, JsonObject mappingProfileJson, JsonObject jobProfileJson, Map<String, String> params) {
     ExportRequest exportRequest = request.mapTo(ExportRequest.class);
     MappingProfile mappingProfile = mappingProfileJson.mapTo(MappingProfile.class);
+    JobProfile jobProfile = jobProfileJson.mapTo(JobProfile.class);
```

Comment:
Yes, I think it's better to populate job profile on the controller level as you proposed instead of passing one more additional parameter to InputManager

Reply:
updated